pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -414,6 +416,11 @@ func (s *Service) decryptSecrets(ctx context.Context, settings map[string]any) (
 
 func (s *Service) isProviderConfigurable(provider string) bool {
 	_, ok := s.cfg.SSOSettingsConfigurableProviders[provider]
+
+	if provider == "saml" && !s.features.IsEnabledGlobally(featuremgmt.FlagSsoSettingsSAML) {
```

Comment:
`saml` will not be added by default to the `configurable_providers` list, so I don't think we need this check here.

Reply:
removed as discussed